R/utils.R

```diff
@@ -97,7 +97,7 @@ name_dots = function(...) {
   if (any(notnamed)) {
     syms = vapply_1b(dot_sub, is.symbol)  # save the deparse() in most cases of plain symbol
     for (i in which(notnamed)) {
-      tmp = if (syms[i]) as.character(dot_sub[[i]]) else deparse(dot_sub[[i]])[1L]
+      tmp = if (syms[i]) as.character(dot_sub[[i]]) else deparse(dot_sub[[i]], nlines=1)[1L]
```

Comment:
I agree this is good (great find!), but now it's not related to the current PR, would you mind separating this to its own PR/NEWS item/test?

it will make the eventual commit history cleaner. thanks!

Reply:
Agree. done.

Reply:
This PR already contains a bit dirty commit history - can you guys squash it when (and if) merging?

Reply:
Great find Ofek! Yes all PRs are always squashed when merged so no problem there. Will do.